### DISCUSSION TITLE
Master  cleaner   delay ignored

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#924] [Q&A] [answered]
My latest challenge...  Turning on the Polaris does not wait 5 minutes prior to starting the booster pump.  It used to & I don't know why it no longer does.  Config screenshots below.  The filter pump does speed up  to the correct speed however.  What the heck am I doing wrong now?

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/d49f1c11-e6de-4365-ba2e-e94a894f67d5)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/eaafd965-3449-4954-9b29-fdaea293dff3)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/f8562877-dd02-4c5c-a96c-02a7c6166eaa)

[replay (8).zip](https://github.com/tagyoureit/nodejs-poolController/files/14898841/replay.8.zip)

**Comment by tagyoureit** [comment, marked as answer]
The cleaner delay is a delay from when the pool circuit pump initially turns on to give it enough time to prime.  It's set to 300s so if you turn on the pool circuit and then turn on the polaris <5 mins after start it will wait until the pool has been on for 5 mins.  

It looks like your pool came on at 8am so at 3:16pm when the replay happened (or 8:30 when the cleaner is scheduled) the delay would not come into effect.

**Reply by FlaMike** [reply]
Thank you for the explanation & clarification.

I ran the cleaner manually this afternoon as I was brushing the pool.  In the past, I thought there was still a delay before the cleaner started even though the filter pump was running.  Based on your explanation, I was mistaken & all is good vis a vis the cleaner delay under this afternoon's circumstances. Yay!

Thank you for your time & explanation!
